spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/TupleValue.scala

```diff
@@ -20,10 +20,11 @@ object TupleValue {
     new TupleValue(values: _*)
   }
 
-  val TupleValueTypeTag = typeTag[TupleValue]
+  val TypeTag = typeTag[TupleValue]
+  val Symbol = typeOf[TupleValue].asInstanceOf[TypeRef].sym
```

Comment:
Hmm, no. Good catch.  I added these in hope I can access them from the core, but then it turned out japi is not on the classpath. I forgot to remove them.